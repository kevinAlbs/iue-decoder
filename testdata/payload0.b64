ADgAAAAQYQABAAAABWtpABAAAAAEYWFhYWFhYWFhYWFhYWFhYQJ2AAwAAAA0NTctNTUtNTQ2MgAA