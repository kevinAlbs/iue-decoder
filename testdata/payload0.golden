BlobSubtype=0
Algorithm=AEAD_AES_256_CBC_HMAC_SHA_512-Random ("a":1)
KeyUUID=61616161616161616161616161616161 ("ki":{"$binary":{"base64":"YWFhYWFhYWFhYWFhYWFhYQ==","subType":"04"}})
Value="457-55-5462" ("v":"457-55-5462")
